src/wallet/coinselection.cpp

```diff
@@ -68,7 +68,7 @@ struct {
  *
  * waste = selectionTotal - target + inputs × (currentFeeRate - longTermFeeRate)
  *
- * The algorithm uses two additional optimizations. A lookahead (tk) keeps track of the total value of
```

Comment:
In 6ff6e98000792f715324c513fd9eb83440929d0e "coinselection: Track effective_value lookahead"

This should be squashed into 9580c0e49030f13fe57603521b59b96e6e50cf82 "coinselection: rewrite BnB in CoinGrinder-style"